index.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+/**
+ * Reads .env configs from your app's `configs/` directory and loads them into
+ * the process environment (via dotenv). Additionally checks for a `NODE_ENV`
+ * and loads additional configs that match `configs/.env.${NODE_ENV}`.
+ * @module loadenv
+ * @author Ryan Sandor Richards, Anand Patel
+ */
+
+var fs = require('fs');
+var path = require('path');
+var dotenv = require('dotenv');
+var eson = require('eson');
+var exists = require('101/exists');
+var clone = require('clone');
+
+/**
+ * Application root directory.
+ * @type {string}
+ */
+var applicationRoot = require('app-root-path').toString();
```

Comment:
Per our discussion, we will enforce the use of `config/` with the possibility of adding an option in the future.